py3-lxml.yaml

```diff
@@ -0,0 +1,47 @@
+# Generated from https://pypi.org/project/lxml/
+package:
+  name: py3-lxml
+  version: 4.9.3
+  epoch: 0
+  description: Powerful and Pythonic XML processing library combining libxml2/libxslt with the ElementTree API.
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3-dev
+      - py3-setuptools
+      - libxml2-dev
+      - libxslt-dev
+      - gcc
+      - cython<3.0.0 # lxml 4.9.3 is incompatible with cython 3+
```

Comment:
Hmm in the past when I tried this it blew up wolfictl iirc

Reply:
The version constraint or the comment at the end of the line?

Reply:
It works locally but not on CI 🤔

Reply:
The version constraint 😞

yeah it was in CI that it but me too.  I wanted goreleaser=1.18 for metacobtroller and had to create a version stream instead

Reply:
So I guess I'll have to create a version stream for Cython? 😭